### PR TITLE
new package: Update example to subject: verb version

### PR DESCRIPTION
**Pull request by ermo**
## Description

With the monorepo, we use the `worklog.py` script and the GH UI, which shows the git summary for each commit at a glance.

Therefore, the git summaries in commits should be as informative as possible so people don't have to go chasing the individual commit logs for relevant version information when looking for specific updates or summarising the weekly development churn in the forum sync notes.

This PR changes the old (and no longer useful) "Initial inclusion" git summary part of the commit message to a more useful "packagename: Add at version" summary.

Note that we already require "packagename: Update to version" for the same reason. This PR also expands on this by adding a section with other useful git summary commit message examples related to common package update scenarios.

### CODE REVIEW

File: docs/packaging/updating-an-existing-package.md
Comment (malfisya) [inline docs/packaging/updating-an-existing-package.md]: I don't think we have documentation for README.md and monitoring.yml yet. Would it more prudent to use MAINTAINERS.md as an example?

File: docs/packaging/updating-an-existing-package.md
Comment (malfisya) [inline docs/packaging/updating-an-existing-package.md:181]: I don't think history of the term is relevant here. It also adds unnecessary jargon. Please remove.

File: docs/packaging/updating-an-existing-package.md
Comment (malfisya) [inline docs/packaging/updating-an-existing-package.md:183]: Can you reprhase this sentence with minimal jargon (git, shortlogs, at-a-glance) ?

**Review by malfisya — CHANGES_REQUESTED** [review]

File: docs/packaging/updating-an-existing-package.md
Comment (ermo) [reply docs/packaging/updating-an-existing-package.md:181]: When we introduced it in serpent, people asked "what does NFC stand for?!", because it is a common abbreviation for "No F...ing Clue", hence I added it for context.

File: docs/packaging/updating-an-existing-package.md
Comment (ermo) [reply docs/packaging/updating-an-existing-package.md:183]: Reads fine to me. Will need someone more familiar with the style guide to chip in here.

File: docs/packaging/updating-an-existing-package.md
Comment (malfisya) [reply docs/packaging/updating-an-existing-package.md:181]: Okay, how about swap the senteces. Definition comes first then the context.

File: docs/packaging/updating-an-existing-package.md
Comment (malfisya) [inline docs/packaging/updating-an-existing-package.md:179]: I don't think we have documentation for README.md and monitoring.yml yet. Would it more prudent to use MAINTAINERS.md as an example?

**Review by malfisya — CHANGES_REQUESTED** [review]

File: docs/packaging/updating-an-existing-package.md
Comment (EbonJaeger) [inline docs/packaging/updating-an-existing-package.md:170]: It might be better to have multiple subheadings for the above section. Something like:

```md
# Commit message formatting

There should be a meaningful summary line (which starts with the package name), a blank line, and then the rest of the commit message.

## Package update

{text from existing "commit message format" section}

## Other package change
```

File: docs/packaging/updating-an-existing-package.md
Comment (EbonJaeger) [reply docs/packaging/updating-an-existing-package.md:183]: If we go with my suggestion above about heading hierarchy, I would like this bit to be in the above "Commit message format" section. Something like "We use a git commit message format to enable quick and efficient parsing of git commit messages."

This does a couple of things:

1. Sets the reasoning up front
2. Immediately introduces the idea of readable commit messages
3. Reduces jargon; the reader doesn't need to know what a shortlog is
4. Makes the language more inclusive; Solus staff aren't the only people that might want to read commit messages

File: docs/packaging/updating-an-existing-package.md
Comment (EbonJaeger) [reply docs/packaging/updating-an-existing-package.md:179]: We do actually have an article for `monitoring.yml` :)

https://help.getsol.us/docs/packaging/monitoring.yml

**Review by EbonJaeger — CHANGES_REQUESTED** [review]
This is a good start. I think we can improve it further, though. :)

**Review by davidjharder — COMMENTED** [review]
I don't have much to add here. 
* I agree with the direction Evan wants to go.
* I would advise against the term shortlog, we don't use it anywhere. I know "title" and "description" are not the correct terms for what is actually all part of the commit message, but people will understand them more intuitively, and github uses those terms. 
* We should *not* link to "tooling central documentation", a repository our users are not familiar with and we do not control. We could steal that guidance and put it on our Git practices page later

A more heretical bit of advice would be to fix exactly one line with this PR to address the precise issue that kicked this all off. I would be fine with that. Your call ermo. The longer this PR is open, the more changes will suggest themeselves

**Comment by TraceyC77** [comment]
I agree with David's points.

**Comment by ermo** [comment]
@davidjharder , @TraceyC77 , @EbonJaeger 

Thank you all for your (excellent!) suggestions.

I've essentially achieved what I wanted with this, which is to shine a light on how the workflow and examples we show new (and returning) packagers will help make the monorepo commit shortlogs more useful to Solus Staff overall.

I have my hands full with Serpent now coming properly online and simply can't afford to give this the copy-editing time it needs in order to be of high enough quality to meet your stands AND be easy to read.

Therefore, I am of the opinion that this really ought to be taken over by the comms/doc team, as you are clearly better at judging what users need here?

Feel free to close this and use it as a basis for a better worded PR?

File: docs/packaging/updating-an-existing-package.md
Comment (EbonJaeger) [inline docs/packaging/updating-an-existing-package.md:186]: We can use a relative link here:

```suggestion
- Updating a README, [monitoring.yml file](monitoring.yml.md) or other stuff that does not directly _change_ the package, meaning the package release number didn't need a bump:
```

File: docs/packaging/updating-an-existing-package.md
Comment (EbonJaeger) [inline docs/packaging/updating-an-existing-package.md:117]: We like Oxford commas :smiley: 

```suggestion
With the switch to a single large package git repository, having a regular, easy-to-read, and standardized commit message format helps enable quick and efficient "at-a-glance" parsing of git commit messages for packagers and Staff alike.
```

**Review by EbonJaeger — CHANGES_REQUESTED** [review]
I think this is almost there! This structure is much better and easier for the reader to get the specific information they want.

**Review by EbonJaeger — APPROVED** [review]
I have no further changes for this PR. Thanks for making this section better!

Regarding @davidjharder 's point about linking to the tooling center docs, I think that's out of scope for this PR and can be addressed by someone later on; the link was already present before this was opened.

**Pull request merged**
